docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+
+services:
+  web:
+    build: .
+    command: python index.py
+    volumes:
+      - ./app/:/usr/src/app/
+    ports:
+      - 8050:8050
```

Comment:
You're sure you don't have to put this into quotes like this `"8050:8050"` ? I have a doubt.

Reply:
I had tried on my end and encountered no issue. But it seems that this can lead to troubles for ports lower than 60: https://stackoverflow.com/questions/58810789/quotes-on-docker-compose-yml-ports-make-any-difference

Just made the modification to avoid troubles later, thanks for catching that up :pray: